incubator/hnc/hack/hnc.yaml

```diff
@@ -0,0 +1,23 @@
+# For PoC purpose only.
+# Will be moved to https://github.com/kubernetes-sigs/krew-index after hnc has it's release.
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: hnc
+spec:
+  shortDescription: Implement hierarchy concept on namespaces.
+  description: |
+    Hierarchical Namespace Controller (HNC).
+  version: "v0.1.0"
```

Comment:
Suggest: replace `"v0.1.0"` with something obviously illegal like ` --- will be populated by make krew-build --- `

Reply:
i see your point. i change `version`, `platform.uri`, and `sha256 `. ptal